### PR TITLE
Wrapped live share code in liveShare namespace

### PR DESCRIPTION
**Pull request by huntj88**
## Description
This is primarily a cleanup PR to align the liveShare code with the rest of the code. Other features are wrapped in their own namespace, which was never done for Live Share.

LiveShareHost is still outside the namespace. Moving that inside the namespace would be a breaking change for LiveShare users.

removed @beta comments, updated doc comments.

### Validation performed:

1. Tested LiveShare apps with teams-js changes

### Unit Tests added: no

No logic changes were introduced, just aligning code patterns.

### CODE REVIEW

File: packages/teams-js/src/public/liveShareHost.ts [hunk 1]
```diff
@@ -3,162 +3,173 @@ import { ensureInitialized } from '../internal/internalAPIs';
...
-}
-
-/**
- * @hidden
- * State of the current Live Share sessions backing fluid container.
- *
- * @beta
- */
-export enum ContainerState {
+export namespace liveShare {
```
Comment (ryanbliss) [inline packages/teams-js/src/public/liveShareHost.ts:6]: Add comment: 
```ts
/**
 * APIs involving Live Share, a framework for building real-time collaborative apps.
 * For more information, visit https://aka.ms/teamsliveshare
 * 
 * @see LiveShareHost
 */
```

File: packages/teams-js/src/public/liveShareHost.ts [hunk 2]
```diff
@@ -3,162 +3,173 @@ import { ensureInitialized } from '../internal/internalAPIs';
...
   /**
-   * The call to `LiveShareHost.setContainerId()` successfully created the container mapping
-   * for the current Live Share session.
+   * Meeting Roles.
```
Comment (ryanbliss) [inline packages/teams-js/src/public/liveShareHost.ts:8]: Replace comment:
```ts
/**
 * The meeting roles of a user.
 * Used in Live Share for its role verification feature.
 */
```

File: packages/teams-js/src/public/liveShareHost.ts [hunk 3]
```diff
@@ -3,162 +3,173 @@ import { ensureInitialized } from '../internal/internalAPIs';
...
+    /**
+     * Organizer role.
+     */
+    organizer = 'Organizer',
+  }
 
   /**
-   * A container mapping for the current Live Share Session already exists and should be used
-   * when joining the sessions Fluid container.
+   * State of the current Live Share sessions backing fluid container.
```
Comment (ryanbliss) [inline packages/teams-js/src/public/liveShareHost.ts:30]: Replace comment:
```ts
/**
 * State of the current Live Share session's Fluid container
 */
```

File: packages/teams-js/src/public/liveShareHost.ts [hunk 4]
```diff
@@ -3,162 +3,173 @@ import { ensureInitialized } from '../internal/internalAPIs';
...
+     * for the current Live Share session.
+     */
+    added = 'Added',
 
-  /**
-   * The call to `LiveShareHost.setContainerId()` failed to create the container mapping due to
-   * another client having already set the container ID for the current Live Share session.
-   */
-  conflict = 'Conflict',
+    /**
```
Comment (ryanbliss) [inline packages/teams-js/src/public/liveShareHost.ts:39]: Replace comment:
```ts
/**
 * A container mapping for the current Live Share session already exists.
 * This indicates to Live Share that a new container does not need be created.
 */
```

File: packages/teams-js/src/public/liveShareHost.ts [hunk 5]
```diff
@@ -3,162 +3,173 @@ import { ensureInitialized } from '../internal/internalAPIs';
...
+     * when joining the sessions Fluid container.
+     */
+    alreadyExists = 'AlreadyExists',
 
-  /**
-   * A container mapping for the current Live Share session doesn't exist yet.
-   */
-  notFound = 'NotFound',
-}
+    /**
```
Comment (ryanbliss) [inline packages/teams-js/src/public/liveShareHost.ts:45]: Replace comment:
```ts
/**
 * The call to `LiveShareHost.setContainerId()` failed to create the container mapping.
 * This happens when another client has already set the container ID for the session.
 */
```

File: packages/teams-js/src/public/liveShareHost.ts [hunk 6]
```diff
@@ -3,162 +3,173 @@ import { ensureInitialized } from '../internal/internalAPIs';
...
- *
- * @beta
- */
-export interface IFluidContainerInfo {
-  /**
-   * State of the containerId mapping.
-   */
-  containerState: ContainerState;
+    /**
+     * A container mapping for the current Live Share session doesn't exist yet.
```
Comment (ryanbliss) [inline packages/teams-js/src/public/liveShareHost.ts:52]: Replace comment:
```ts
/**
 * A container mapping for the current Live Share session does not yet exist.
 * This indicates to Live Share that a new container should be created.
 */
```

File: packages/teams-js/src/public/liveShareHost.ts [hunk 7]
```diff
@@ -3,162 +3,173 @@ import { ensureInitialized } from '../internal/internalAPIs';
...
+     */
+    notFound = 'NotFound',
+  }
 
   /**
-   * ID of the container to join for the meeting. Undefined if the container hasn't been
-   * created yet.
+   * Returned from `LiveShareHost.get/setFluidContainerId()` to specify the container mapping for the
```
Comment (ryanbliss) [inline packages/teams-js/src/public/liveShareHost.ts:58]: Replace comment:
```ts
/**
 * Returned from `LiveShareHost.getFluidContainerId()` and `LiveShareHost.setFluidContainerId`.
 * This response specifies the container mapping information for the current Live Share session.
 */
```

File: packages/teams-js/src/public/liveShareHost.ts [hunk 8]
```diff
@@ -3,162 +3,173 @@ import { ensureInitialized } from '../internal/internalAPIs';
...
-   */
-  retryAfter: number;
-}
+    /**
+     * If true, the local client should create the container and then save the created containers
+     * ID to the mapping service.
+     */
+    shouldCreate: boolean;
+
+    /**
```
Comment (ryanbliss) [inline packages/teams-js/src/public/liveShareHost.ts:79]: Replace comment:
```ts
/**
 * If `containerId` is undefined and `shouldCreate` is false, the container isn't ready but another
 * client is creating it. In this case, the local client should wait the specified amount of time before
 * calling `LiveShareHost.getFluidContainerId()` again.
 */
```

File: packages/teams-js/src/public/liveShareHost.ts [hunk 9]
```diff
@@ -3,162 +3,173 @@ import { ensureInitialized } from '../internal/internalAPIs';
...
+  export interface IClientInfo {
+    /**
+     * Teams userId associated with clientId
+     */
+    userId: string;
+    /**
+     * Meeting roles associated with clientId
+     */
+    roles: UserMeetingRole[];
+    /**
```
Comment (ryanbliss) [inline packages/teams-js/src/public/liveShareHost.ts:132]: Replace comment:
```ts
/**
 * The user's display name associated for a given `clientId`.
 * If this returns as `undefined`, the user may need to update their host client.
 */
```

File: packages/teams-js/src/public/liveShareHost.ts [hunk 10]
```diff
@@ -3,162 +3,173 @@ import { ensureInitialized } from '../internal/internalAPIs';
...
+   * Returned from `LiveShareHost.getClientInfo()` to specify the client info for a
+   * particular client in a Live Share session.
    */
-  tenantId: string;
+  export interface IClientInfo {
+    /**
+     * Teams userId associated with clientId
+     */
+    userId: string;
+    /**
```
Comment (ryanbliss) [inline packages/teams-js/src/public/liveShareHost.ts:128]: Replace comment:
```ts
/**
 * User's meeting roles associated with a given `clientId`
 */
```

File: packages/teams-js/src/public/liveShareHost.ts [hunk 11]
```diff
@@ -3,162 +3,173 @@ import { ensureInitialized } from '../internal/internalAPIs';
...
-  displayName?: string;
+  export function isSupported(): boolean {
+    return ensureInitialized(runtime, FrameContexts.meetingStage, FrameContexts.sidePanel) &&
+      runtime.supports.interactive
+      ? true
+      : false;
+  }
 }
 
 /**
```
Comment (ryanbliss) [inline packages/teams-js/src/public/liveShareHost.ts:162]: Replace comment:
```ts
/**
 * Live Share host implementation for connecting to real-time collaborative sessions.
 * Designed for use with the `LiveShareClient` class in the `@microsoft/live-share` package.
 * Learn more at https://aka.ms/teamsliveshare
 * 
 * @remarks
 * The APIs in this class are not intended to be used directly by your app.
 * The `LiveShareClient` class from Live Share uses this to join/manage the session.
 */
```

File: packages/teams-js/src/public/liveShareHost.ts [hunk 12]
```diff
@@ -266,34 +259,16 @@ export class LiveShareHost {
    *
```
Comment (ryanbliss) [inline packages/teams-js/src/public/liveShareHost.ts:269]: Replace comment:
```ts
/**
 * Factories a new `LiveShareHost` instance for use with the `LiveShareClient` class
 * in the `@microsoft/live-share` package.
 * 
 * @remarks
 * `app.initialize()` must first be called before using this API. 
 * This API can only be called from `meetingStage` or `sidePanel` contexts.
 */
```

**Comment by huntj88** [comment]
closing, switching source branch for non forked branch
https://github.com/OfficeDev/microsoft-teams-library-js/pull/1803

**Pull request closed without merging**